movie-browser/src/App.js

```diff
@@ -21,13 +24,27 @@ class App extends Component {
       });
   }
 
+  handleToggleMovies = () => {
+    if (!this.state.toggleMovies) {
+      this.buttonText = 'Show Movies';
+    } else {
+      this.buttonText = 'Hide Movies';
+    }
+
+    this.setState({
+      toggleMovies: !this.state.toggleMovies,
+      buttonText: this.buttonText,
```

Comment:
```suggestion
      buttonText: toggleMovies ? 'Hide movies' : 'Show Movies', 
```

Reply:
this was just a suggestion that you could use the ternary if.

`condition ? true_statement : false_statement`

Reply:
Thanks for the suggestion! This a lot more readable